### PR TITLE
Prac 09 feedback request

### PR DESCRIPTION
**Pull request by ellBasso**
@chanteneviljoen Please check formatting, naming and logic, Thanks.

### CODE REVIEW

File: prac_09/sort_files_1.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import shutil
+import os
+
+
+def main():
```
Comment (chanteneviljoen) [inline prac_09/sort_files_1.py:7]: Remember docstrings

File: prac_09/sort_files_2.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import shutil
+import os
+
+
+def main():
```
Comment (chanteneviljoen) [inline prac_09/sort_files_2.py:7]: Remember docstrings

**Comment by chanteneviljoen** [comment]
I could not identify any major issues, your work looks good! Just remember to add in your docstrings :)

**Comment by ellBasso** [comment]
> I could not identify any major issues, your work looks good! Just remember to add in your docstrings :)

Thanks for the feedback :)

**Pull request merged**